pkg/orchestrator/pod.go

```diff
@@ -112,16 +112,25 @@ func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterNa
 
 // ScrubContainer scrubs sensitive information in the command line & env vars
 func ScrubContainer(c *v1.Container, scrubber *DataScrubber) bool {
-	// scrub command line
-	scrubbedCmd, changed := scrubber.ScrubSimpleCommand(c.Command)
-	c.Command = scrubbedCmd
 	// scrub env vars
 	for e := 0; e < len(c.Env); e++ {
 		if scrubber.ContainsSensitiveWord(c.Env[e].Name) {
 			c.Env[e].Value = redactedValue
-			changed = true
```

Comment:
why do we need to remove this?

Reply:
Because of the ineffectual check of go
```
 ineffassign ./pkg ./cmd   

/go/src/github.com/DataDog/datadog-agent/pkg/orchestrator/pod.go:120:4: ineffectual assignment to changed
```

but I initially introduced that value to avoid compiler optimisation in the benchmarking. We did not have that return value before.
But this is unnecessary, hence, I removed this again 👍